esp/applications/dfs/ldap_authorization_map.yaml

```diff
@@ -0,0 +1,5 @@
+ldap:
+  root_access:
+    resource: WsDfsAccess
+    required: Read
+    description: Base access to EclWatch
```

Comment:
@jakesmith if it's needed, the auth resource map now goes here.

```
   resource_map:
      ws_dfsservice:
         Feature:
         -  path: WsDfsAccess
            resource: WsDfsAccess
            description: Base access to DFS services
```

Reply:
(edited to match the auth feature tag used in your esdl ecm file.

Reply:
Also I just noticed that in the environment xml related changes for the baremetal deployment I think you used the auth feature tag "WsDfsAccess".  The name has to match what you reference in anywhere in your esdl ecm file, or c++ code.

In this case:

```
auth_feature("DfuAccess:READ")
```

Reply:
yes, ok, will need correcting in other PR

Reply:
See : https://github.com/hpcc-systems/HPCC-Platform/pull/15686

Reply:
> @jakesmith if it's needed, the auth resource map now goes here.
> 
> ```
>    resource_map:
>       ws_dfsservice:
>          Feature:
>          -  path: DfsAccess
>             resource: DfsAccess
>             description: Base access to DFS services
> ```

Will correct this once https://github.com/hpcc-systems/HPCC-Platform/pull/15686 is merged